report/renderer/topbar-features.js

```diff
@@ -302,13 +297,15 @@ export class TopbarFeatures {
       categoriesAboveTheMiddle.length > 0 ? categoriesAboveTheMiddle.length - 1 : 0;
 
     // Category order matches gauge order in sticky header.
-    const gaugeWrapperEls = this.stickyHeaderEl.querySelectorAll('.lh-gauge__wrapper');
+    const gaugeWrapperEls =
+      this.stickyHeaderEl.querySelectorAll('.lh-gauge__wrapper, .lh-fraction__wrapper');
     const gaugeToHighlight = gaugeWrapperEls[highlightIndex];
-    const origin = gaugeWrapperEls[0].getBoundingClientRect().left;
-    const offset = gaugeToHighlight.getBoundingClientRect().left - origin;
+    for (const gauge of gaugeWrapperEls) {
+      gauge.classList.remove('lh-gauge--highlight');
+    }
+    gaugeToHighlight.classList.add('lh-gauge--highlight');
 
     // Mutate at end to avoid layout thrashing.
-    this.highlightEl.style.transform = `translate(${offset}px)`;
```

Comment:
The above comment applied to this line. Is there not layout thrashing with these changes? I guess not because you made CSS changes to replace the manual transform setting?

Reply:
Can you confirm w/ the performance panel that scrolling doesn't result in tons of style recalc?

Reply:
There is some small style updates, but those existed before this change.

Reply:
Paul reminded me... before, this highlighter was animated between these states. Now that's lost.

idk if we care tbh

Reply:
also we noticed that clicking the gauges at the top doesnt always update highlighter correctly, viewport size seems to make it vary

Reply:
The highlight is based on whichever category is in the center of the viewport. For snapshot, performance category can be so small that it will never be in the center.

Honestly, one solution could be to make Accessibility the first category for snapshot reports. Performance isn't important enough to be first for snapshot reports IMO.